rocky/rocky/views/ooi_list.py

```diff
@@ -35,8 +36,11 @@ def get_context_data(self, **kwargs):
 
         context["types_display"] = self.get_ooi_types_display()
         context["object_type_filters"] = self.get_ooi_type_filters()
+        context["mandatory_fields"] = get_mandatory_fields(self.request, params=["observed_at"])
         context["select_oois_form"] = SelectOOIForm(
-            context.get("ooi_list", []), organization_code=self.organization.code
+            context.get("ooi_list", []),
+            organization_code=self.organization.code,
+            mandatory_fields=context["mandatory_fields"],
```

Comment:
I dont think this field should be mandatory, when left blank XTDB just defaults to 'now', which should be de default.

Reply:
Yes, that's how it works. I don't know why this is called "mandatory_fields", but that's the name of the param I found on other places where this was implemented...